test/container/mxnet/Dockerfile

```diff
@@ -63,69 +59,39 @@ RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSIO
 RUN ln -s $(which ${PYTHON}) /usr/local/bin/python
 
 RUN ${PIP} --no-cache-dir install --upgrade \
-    pip==${PIP_VERSION} \
+    pip \
     setuptools
 
 WORKDIR /
 
-COPY mxnet/sagemaker_mxnet_serving_container.tar.gz /sagemaker_mxnet_serving_container.tar.gz
+COPY sagemaker_mxnet_inference.tar.gz /sagemaker_mxnet_inference.tar.gz
 
 RUN ${PIP} install --no-cache-dir \
     ${MX_URL} \
+    git+git://github.com/dmlc/gluon-nlp.git@v0.9.0 \
     mxnet-model-server==$MMS_VERSION \
     keras-mxnet==2.2.4.1 \
     numpy==1.17.4 \
     onnx==1.4.1 \
-    /sagemaker_mxnet_serving_container.tar.gz \
- && rm /sagemaker_mxnet_serving_container.tar.gz
+    /sagemaker_mxnet_inference.tar.gz \
+ && rm /sagemaker_mxnet_inference.tar.gz
 
 # This is here to make our installed version of OpenCV work.
 # https://stackoverflow.com/questions/29274638/opencv-libdc1394-error-failed-to-initialize-libdc1394
 # TODO: Should we be installing OpenCV in our image like this? Is there another way we can fix this?
 RUN ln -s /dev/null /dev/raw1394
 
-##################################
-# download models to model_store
```

Comment:
Are models downloaded anywhere else?

If models are not downloaded here, another way to make it available for the test is to download the model before any load model requests are sent

Reply:
Re-added the commands to download models in the `Dockerfile`.